src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
```

Comment:
Using a hardcoded index (3) to access the separator in `fromFormat` assumes that the separator is always at this position. This may not be the case if the format changes. Consider passing the separator as a separate parameter or determining its position dynamically.